src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -148,14 +149,16 @@ public function onSubmit(FormEvent $event)
             throw new UnexpectedTypeException($data, 'array or (\Traversable and \ArrayAccess)');
         }
 
-        if ($this->deleteEmpty) {
-            $previousData = $event->getForm()->getData();
+        if ($entryFilter = $this->deleteEmpty) {
+            $previousData = $form->getData();
             foreach ($form as $name => $child) {
+                /* @var $child FormInterface */
```

Comment:
See PSR-5: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#3-definitions

About the foreach, though:

> This Standard does not cover this specific instance as a foreach statement is not considered to be a "Structural Element" but a Control Flow statement.

but still:

> It is RECOMMENDED to precede a "Structural Element" with a DocBlock where it is defined and not with each usage.